### DISCUSSION TITLE
maximum number of users reached

### DISCUSSION THREAD

**Discussion by faha1999** [lowlighter/metrics#1133] [Q&A] [answered]
An error occurred while generating your metrics :( Service Unavailable: maximum number of users reached, only cached metrics are available

How to solve this issue?

**Comment by Maou-Shimazu** [comment]
I dont know about the issue internally so I cannot provide a solution, but I am aware that the beta site works as of right now [https://beta-metrics.lecoq.io/](https://beta-metrics.lecoq.io/)

**Comment by lowlighter** [comment, marked as answer]
I removed the maximum users limit on both beta-metrics.lecoq.io and metrics.lecoq.io so now you shouldn't encounter this error message anymore 👍
